src/test/kotlin/CrashReaderIntegrationTest.kt

```diff
@@ -67,19 +68,21 @@ class CrashReaderIntegrationTest {
     }
 
     @ParameterizedTest
-    @ResourceFilesSource("crashes/jvm")
+    @ResourceFilesSource("crashes/jvm", ignoredNameSuffixes = ["-parsed"])
     fun shouldRunAllJvmCrashes(crashFile: File) {
-        val name = crashFile.name
-        val parts = name.split("-")
-        val isModded = parts[0] == "true"
-        val expectedCode = parts[1]
+        val expectedParsedCrashString = File(crashFile.parent, crashFile.nameWithoutExtension + "-parsed.txt")
```

Comment:
Uses `.txt` extension even though these files contain JSON because that is a test implementation detail.